### ISSUE TITLE
Only works with Chrome

### ISSUE THREAD

**Issue by crushv** [droibit/flutter_custom_tabs#2]
Custom tabs should work with chromium and firefox but this implementation does not.

I'm looking into how this could be fixed.

**Comment by crushv** [comment]
After looking into the source it's obvious that it's caused by https://github.com/droibit/CustomTabsLauncher#priority-of-chrome

https://github.com/droibit/CustomTabsLauncher/issues/1

**Comment by droibit** [comment]
Yes. I'm going to consider updating`CustomTabsLauncher` library.

**Comment by droibit** [comment]
@crushv   
I just published latest v`0.4.0` plugin.  
If you want to launch Firefox as CustomTabs, specify the package name(s) to the newly added option [`CustomTabsOptions.extraCustomTabs`](https://pub.dartlang.org/documentation/flutter_custom_tabs/latest/flutter_custom_tabs/CustomTabsOption/extraCustomTabs.html).

example:
```dart
await launch(
  // ...
  option: new CustomTabsOption(
    // ...
    extraCustomTabs: <String>[
      // ref. https://play.google.com/store/apps/details?id=org.mozilla.firefox
      'org.mozilla.firefox',
    ],        
  ),
);
```

**Comment by XiNiHa** [comment]
Why is it only considered as `fallback`? Even if I set the default browser to something other than Chrome, it looks like it always prioritizes Chrome when it's installed. Is it possible to follow the users' default browser preference? Apps like Twitter does this without any problem.
